.circleci/config.yml

```diff
@@ -1,11 +1,20 @@
 version: 2.1
 
+orbs: 
+  node: circleci/node@5.0.2
 jobs:
   build:
     working_directory: ~/openreview-expertise
     docker:
       - image: ubuntu:18.04
       - image: redislabs/redisai
+      - image: circleci/mongo:4.4.5
```

Comment:
Let's replace this to be mongo v5 😄

Reply:
Any version in particular that you had in mind or just replace it with `circleci/mongo:5.0`?

Reply:
Any version of 5.0.x should be fine.